### DISCUSSION TITLE
stm32F722ze, ValueError: Pin(PC13) doesn't exist

### DISCUSSION THREAD

**Discussion by ghost** [micropython/micropython#16148] [STM32 / Pyboard]
import pyb

def main():
    #configure pb0 as an output pin push-pull
    green_led = pyb.Pin('D33', mode =pyb.Pin.OUT_PP)
    
    #configure PC13 as input pin with pull-down
    button = pyb.Pin('PC13', mode = pyb.Pin.IN, pull = pyb.Pin.PULL_DOWN)
    
    
    
    while True:
        
        if button.value() == 0:
            green_led.value(1)
        else:
            green_led.value(0)
        
    
main()

I am using stm32f722ze and trying to write a program to control led using push button but I always says value error. With the help of user manual figure out the arduino mapping for PC0 but cannot find for PC13. Is there any way I will be able to execute this program. I am using Thonny IDE. Thanks

**Comment by davefes** [comment]
What dev board is your STM32 on?  I use the WeAct BlackPill STM32F411 and PC13 is used as the anti-tamper pin.  I failed when I accidentally tried to use it for something else.

**Reply by ghost** [reply]
I am using STM NUCLEO-F722ZE

**Reply by davefes** [reply]
Actually the WeAct board has the blue LED on PC13 ... maybe that is what prevented me from using it as a GPIO.

**Reply by davefes** [reply]
Check your schematic to see if they use PC13 for some on-board reason.
